lib/DataObject/GridColumnConfig/Operator/TranslateValue.php

```diff
@@ -22,9 +22,6 @@
 
 class TranslateValue extends AbstractOperator
 {
-    /**
-     * @var Translator
```

Comment:
change to TranslatorInterface?

Reply:
IMHO, TranslatorInterface is not enough as there is no setLocale method. Changed it to Translator. 
Besides that, all operators should be marked as described here
https://github.com/pimcore/pimcore/issues/7189

Reply:
@weisswurstkanone then why not `TranslatorInterface|LocaleAwareInterface`?

Reply:
will try, btw all those classes should be marked final and internal anyway. see #7189

Reply:
done